content/storage-history/history.textile

```diff
@@ -0,0 +1,633 @@
+---
+title: History
+meta_description: "Learn about accessing message history with the History and Rewind features"
+languages:
+ - javascript 
+ - java 
+ - python 
+ - php 
+ - ruby 
+ - nodejs 
+ - objc
+ - swift
+ - go 
+ - csharp
+---
+
+History refers to the storing and accessing of previously sent messages, Ably has two approaches to history: channel history and presence history.
+
+History stores message events published to a channel. You can combine both "live" and longer term persisted history up to the point a realtime channel was attached. If "persisted history is enabled":/storage-history/storage messages will typically be stored for 24 or 72 hours, otherwise Ably retains the last two minutes of message history. Read more about channels in general in the "channel section":/channels.
+
+The event history on a channel such as enter, update, and leave events is stored in "presence":/presences-occupancy.
+
+h2(#history-rewind). History vs Rewind
+
+There are two ways of accessing your channel history: History or Rewind. Both History and Rewind need the channel to have "persistence enabled":/storage-history/storage.
+
+h3(#history). History
+
+The History feature allows you to retrieve any message published within your "persistence timeframe":/storage-history/storage. 
+
+History is useful for cases where you need to access messages published beyond the most recent, for example:
+
+* Chat applications where the entirety of a chat should be available to new joiners
+* Seeing the history of periodically sent data, such as weather apps that allow you to see historical weather 
+* IoT devices that care about historical statuses
+
+h3(#rewind). Rewind
+
+The rewind parameter allows you to specify where to start an attach from at the time of attaching to a channel.
+
+You can specify either:
+
+* A given number of messages.
+* A point in time in the past, as a time interval.
+
+Rewind is also the only way return the last event published to a channel persisted for 365 days if "persist last message is enabled":/storage-history/storage#persist-last-message.
+
+Note that Rewind is only available with the realtime interface.
+
+Rewind is particularly useful in the following scenarios:
+
+* Realtime chat applications that only need to show the most recent context, not an entire history
+* A channel providing periodic data, such as weather apps that only provide the immediate weather
+* IoT devices that only care about the current state
+
+h2(#retrieve-channel). Retrieve channel history
+
+You can retrieve channel history by using the @history@ method. 
```

Comment:
We should link to the method when referencing it here please. Also would write it as `history()`.